presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-hdfs-impersonation/hive-hdfs-impersonation.properties

```diff
@@ -0,0 +1,8 @@
+connector.name=hive-cdh5
```

Comment:
If you are going to create separate folder for each environment in `environment-specific-catalogs` than you probably don't need to duplicate the environment name in `hive.properties` file name. What do you think?